fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -59930,7 +59942,7 @@ creation_date: 2009-04-03T05:28:18Z
 [Term]
 id: FBbt:00007339
 name: serotonergic SP1 neuron
-def: "Serotonergic neuron whose cell body is located in the anterodorsal brain. There is a cluster of neurons in each hemisphere. In adults, there are three neurons per cluster, with two of these being generated during metamorphosis." [FlyBase:FBrf0048551, FlyBase:FBrf0158877]
+def: "Serotonergic neuron whose cell body is located in the anterodorsal brain, in a cluster anterior to the serotonergic SP2 neurons." [FlyBase:FBrf0048551, FlyBase:FBrf0158877]
```

Comment:
Wondering why details stripped out.  I think it's fine to include info about numbers per cluster and when generated as a gloss after the main part of the def lays out minimal necessary and sufficient conditions,

Reply:
Although - I see you've kept this elsewhere so perhaps just your judgment that this is not well supported or on wrong term?